binderhub/builder.py

```diff
@@ -368,7 +368,7 @@ async def get(self, provider_prefix, _unescaped_spec):
         else:
             push_secret = None
 
-        BuildClass = FakeBuild if self.settings.get('fake_build') else Build
+        BuildClass = FakeBuild if self.settings.get('fake_build') else self.settings.get('build_class')
```

Comment:
Do we still need the `fake_build` setting or can we set `c.BinderHub.build_class` instead in:
https://github.com/jupyterhub/binderhub/blob/1957bfb0df4bffd2cd31d6915a6d132426c614ae/testing/local-binder-mocked-hub/binderhub_config.py#L15
This would also serve to validate this change.

Reply:
That would be great.

Reply:
Guess, I could successfully remove it (plz have a look!).
Do you want me to move the `FakeBuild` class to `testing/local-binder-mocked-hub/binderhub_config.py`?
This would be possible now.

Reply:
@yuvipanda any preference on where `FakeBuild` should be defined? Or can we merge this now?

Reply:
I guess I better should not have asked :stuck_out_tongue_closed_eyes:

Reply:
Two approvals and three weeks later... I'm merging 😁. We can move the class to a different location in another PR.